frontend/src/sections/SponsorsAndPartners/Sponsors/IndividualSponsors/IndividualSponsors.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Image from 'next/image';
+import { AiOutlineLink } from 'react-icons/ai';
+import account from "../../../../../public/account.png"
```

Comment:
Do we need to import from public dir ? Can explore about hosting static assets in nextJS